### DISCUSSION TITLE
Add ability to use query strings for CP search queries

### DISCUSSION THREAD

**Discussion by jacobgraf** [craftcms/cms#8717] [Ideas] [answered]
I would love to be able to deep link into search results in the Control Panel. Our use case is for searching/filtering orders in Craft Commerce, but I could see it being useful for other Control Panel search result deep links.

**Comment by TonyDeStefano** [comment]
Adding to this, it would be amazing to be able to search for an entry, click to the view the entry, hit the back button and see your previous search results.

**Comment by brandonkelly** [comment, marked as answer]
This has been added for Craft 4.4 🎉 (via #12399)
